libs/utils/test.py

```diff
@@ -191,6 +191,12 @@ def get_end_times(self, experiment):
 
         return end_times
 
+    # Defining a default, empty runTest allows test classes to be conveniently
+    # instantiated. Since we run tests via nose, runTest isn't used. Instead all
```

Comment:
Yeah, this comment is pretty unclear clear but it's because when you directly instantiate a `TestCase` it looks for a method called `runTest` (unless you specify a different method, which AFAICT is what `nosetests` does.

Anyway I tried out your suggestion of using `runTest` instead of `init` and it works find so let's do that.